### PR TITLE
Frequency of unique characters of a String

### LINKED ISSUE #4: Frequency of Characters
# Description

Write a program to find the frequency of unique characters in a string and store the result in a text file.

## Language

- [x] C++: @ShivamPorwal02 

## Type of issue

- [x] Feature
- [ ] Bug

### PR DESCRIPTION
**Pull request by YashSaxena9**
# Frequency of characters

The following snippet finds the frequency of all unique readable characters present in a string and writes the frequency count in an output file.

Fixes #4 

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generates no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by kaustubhgupta — APPROVED** [review]

File: snippets/java/charFreqCount.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        int[] chars = new int[127 - 32];      //  readble ASCII characters are from 32 to 126 number
+        for (char c : str) {
+            chars[c - ' ']++;       //  ASCII(32) = ' '
+        }
+
+        //  output to write in file
+        StringBuilder res = new StringBuilder("Frequency of characters in input string is : \n\n");
+        for (int i = 0, N = chars.length; i < N; i++) {
+            if (chars[i] != 0) {    //  if frequency is not 0 then append in result string
+                res.append("character '" + (char)(' ' + i) + "' present :" + chars[i] + " times\n");
```
Comment (sakshamarora1) [inline snippets/java/charFreqCount.md]: I think a proper sentence will look better along with the corresponding changes in the output block. :)
```suggestion
                res.append("The character '" + (char)(' ' + i) + "' occurred " + chars[i] + " times.\n");
```

**Review by sakshamarora1 — CHANGES_REQUESTED** [review]
Thank you for opening a Pull Request :partying_face:!
This looks great! Just a single change :)

**Comment by YashSaxena9** [comment]
Changes are done, please have a look. :)

**Comment by sakshamarora1** [comment]
The output block isn't updated yet. @YashSaxena9
Looks good and will be ready to merge after this. :D

**Comment by YashSaxena9** [comment]
Updated the output block, hope this resolves all problems now. :)

**Review by sakshamarora1 — APPROVED** [review]

**Comment by YashSaxena9** [comment]
> ```Java
> PRs count if:
> Submitted during the month of October AND (
>   The PR is labelled as hacktoberfest-accepted by a maintainer OR
>   Submitted in a repo with the hacktoberfest topic AND (
>     The PR is merged OR
>     The PR has been approved
>   )
> )
> ```
> 

I'm sorry but their's no tag regarding `hacktoberfest-accepted` which is compulsory for a PR to count for Hacktoberfest submission.
Is this repository not counted for Hacktoberfest?

**Comment by sakshamarora1** [comment]
@YashSaxena9 I have added the appropriate label! Thanks for noticing.

**Pull request merged**